### PR TITLE
Adds Code Challenge blog post

### PR DESCRIPTION
**Pull request by duksis**
@RoxasShadow @emmalou could you please add this to your evening reading list? :)

### CODE REVIEW

File: _posts/2015-12-14-code-challenge.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+layout:  post
+author:  hugo_duksis
+categories: Developer
+cover_image: honey01.jpg
+permalink:  code-challenge
+importance: 2
+---
+
+
+Every developer on Honeypot is prescreened before they are accepted onto a batch.
```
Comment (RoxasShadow) [inline _posts/2015-12-14-code-challenge.md]: I guess "they are" is not [correct](http://www.oxfordlearnersdictionaries.com/definition/english/every). Do you agree?

File: _posts/2015-12-14-code-challenge.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+This is beneficial for both sides: for devs, passing the challenge
+grants access to a platform with high quality companies and job offers, and for companies,
+pre-screening simplifies their work.
+
+
+Our first stop for evaluation is activity on relevant public profiles. In cases where talent doesn’t
```
Comment (RoxasShadow) [inline _posts/2015-12-14-code-challenge.md]: "step"?

File: _posts/2015-12-14-code-challenge.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+have any opensource code on github or contributions on other platforms we offer a challenge.
```
Comment (RoxasShadow) [inline _posts/2015-12-14-code-challenge.md]: GitHub

File: _posts/2015-12-14-code-challenge.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+The challenges are delivered through a git repositories hosted on GitHub that contain a simple description of the task.
```
Comment (RoxasShadow) [inline _posts/2015-12-14-code-challenge.md]: I guess we can avoid the "a" in "through a git repositories"

File: _posts/2015-12-14-code-challenge.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+We search for evidence of structured, constructive communication. This indicates a developer
+who is a team player.
+
+ 6. Knowledge of different languages and programming paradigms
+Knowing how different problems are approached in different environments
+expands the scope of possible solutions when a problem is faced. And shows general learning
+abilities.
+
+
+ *Code quelity*
```
Comment (RoxasShadow) [inline _posts/2015-12-14-code-challenge.md]: quality

File: _posts/2015-12-14-code-challenge.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+
+ 1. Does it work? (is executable and behaves as specified)
+Does the solution solve the problem at hand.
+
+ 2. Testing (unit, integration and/or acceptance tests, they pass and cover public interface)
+It is always good tho have someone else to verify that your changes and are delivering the expected
```
Comment (RoxasShadow) [inline _posts/2015-12-14-code-challenge.md]: tho -> to

**Comment by RoxasShadow** [comment]
Left some comments, looks good, gg :+1:

File: _posts/2015-12-14-code-challenge.md (hunk 2 above)
Comment (emmalou) [inline _posts/2015-12-14-code-challenge.md]: where talent "don't" (talent is plural in this instance, oder?)

File: _posts/2015-12-14-code-challenge.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+Our evaluation criteria takes the assumption that code is written once but read multiple
+times. We do not evaluate the challenge results in isolation but rather consider them
+to be an addition to existing profile.
+
+So what do we value? We have divided it in two categories, workflow best practice and code quality.
+The following is an overview of what we are up to at Honeypot:
+
+
+*Workflow best practices*: these general characteristics show a developer’s
+ability to constructive communication, work in teams and think organizationally.
```
Comment (emmalou) [inline _posts/2015-12-14-code-challenge.md]: "show a developer's ability to communicate constructively"

**Comment by duksis** [comment]
wow thanks both of you. Haven't had so many comments on a PR for a while :)
will address all of your comments.

File: _posts/2015-12-14-code-challenge.md [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+Continuous Integration (CI) is a development practice in which developers
+integrate their code into a shared repository several times a day. It allows team members
+to detect problems early.
+
+ 3. Usage of documentation
+Documentation, such as a basic README, describes what the code does,
+how it is set up, why one framework instead of another was chosen. This allows newcomers
+to the project get easier up to speed.
+
+ 4. Usage of automated server setup, tools such as Chef, Ansible, Saltstack, Docker, Vagrant
```
Comment (emmalou) [inline _posts/2015-12-14-code-challenge.md]: USage of automated server setup tools (no comma after setup)

File: _posts/2015-12-14-code-challenge.md (hunk 1 above)
Comment (duksis) [inline _posts/2015-12-14-code-challenge.md]: agree, but than "he is" or "he/she is" not to discriminate anyone

File: _posts/2015-12-14-code-challenge.md [hunk 9]
```diff
@@ -0,0 +1,96 @@
+---
+title:   Code Challenge
+subtitle:
```
Comment (duksis) [inline _posts/2015-12-14-code-challenge.md]: still missing a subtitle

**Pull request merged**